README.md

```diff
@@ -2,7 +2,9 @@
 
 Horreum is a service for storing performance data and regression analysis.
 
-Project website: [https://horreum.hyperfoil.io](https://horreum.hyperfoil.io)
+Please, visit our Project website: [https://horreum.hyperfoil.io](https://horreum.hyperfoil.io)
```

Comment:
Project -> project

Have the URL in a separate section such that it appearss on its own.

For -> for